lib/chef/knife/changelog.rb

```diff
@@ -51,6 +51,12 @@ class Changelog < Knife
         :long => '--update',
         :description => 'Update Berksfile'
 
+      option :allow_errors,
```

Comment:
Do we really need this? Can't we just display an error message for erroneous cookbooks and not raise in any case.

Reply:
Not without at least bumping to a version 2. We would be clearly making a breaking change. I understand the current behaviour has been there from v 0.1.

Reply:
Let's do a v2 then :)